### PR TITLE
Updates to Demo UI

### PR DESCRIPTION
**Pull request by sohamkasar19**
- Changes to the main container of the demo
- Options footer moved to the settings modal

### CODE REVIEW

File: apps/demo-ui/ui/src/App.tsx [hunk 1]
```diff
@@ -75,7 +75,7 @@ export default function App() {
         previousInteractions = [...previousInteractions, chat]
       })
       console.log("Setting previous interactions", previousInteractions)
-      setChatHistory(previousInteractions)
+      setChatHistory(previousInteractions.reverse())
```
Comment (baitsguy) [inline apps/demo-ui/ui/src/App.tsx:78]: Can you just update the order in line 75?

File: apps/demo-ui/ui/src/Chatbox.tsx [hunk 2]
```diff
@@ -1000,52 +1055,91 @@ export const ChatBox = ({ chatHistory, searchResults, setChatHistory, setSearchR
...
+                    
+                    <Container  className={classes.fixedBottomContainer}>
+                        <Group position={!disableFilters && settings.required_filters.length > 0 ? "apart" : 'right'} w='65vw' ml='auto' mr='auto' p='sm' h='3.5em'>
+                            {!disableFilters && settings.required_filters.length > 0 ? <FilterInput settings={settings} filtersInput={filtersInput} setFiltersInput={setFiltersInput} filterError={filterError} setFilterError={setFilterError} /> : null}
+                            
+                            <SearchControlPanel disableFilters={disableFilters} setDisableFilters={setDisableFilters} questionRewriting={questionRewriting} setQuestionRewriting={setQuestionRewriting}
+                                queryPlanner={queryPlanner} setQueryPlanner={setQueryPlanner} chatHistory={chatHistory} setChatHistory={setChatHistory} openSearchQueryEditorOpenedHandlers={openSearchQueryEditorOpenedHandlers} settings={settings}></SearchControlPanel>
+                        </Group>
+                        <Center>
+                        {/* <form onSubmit={handleSubmit} className="input-form" > */}
```
Comment (baitsguy) [inline apps/demo-ui/ui/src/Chatbox.tsx:1111]: can you clean the commented stuff

File: apps/demo-ui/ui/src/ConversationList.tsx [hunk 3]
```diff
@@ -120,6 +120,9 @@ const NewConversationInput = ({ refreshConversations, setErrorMessage, chatInput
     const mobileScreen = useMediaQuery(`(max-width: ${theme.breakpoints.md})`);
     
     const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        if(error) {
```
Comment (baitsguy) [inline apps/demo-ui/ui/src/ConversationList.tsx:123]: can just replace with setError(!error)? same as below

**Review by baitsguy — APPROVED** [review]
Couple comments, let's address those in the follow up PR with filters

File: apps/demo-ui/ui/src/App.tsx (hunk 1 above)
Comment (sohamkasar19) [reply apps/demo-ui/ui/src/App.tsx:78]: Addressed in PR #394

File: apps/demo-ui/ui/src/Chatbox.tsx (hunk 2 above)
Comment (sohamkasar19) [reply apps/demo-ui/ui/src/Chatbox.tsx:1111]: Addressed in PR #394

File: apps/demo-ui/ui/src/ConversationList.tsx (hunk 3 above)
Comment (sohamkasar19) [reply apps/demo-ui/ui/src/ConversationList.tsx:123]: Addressed in PR #394

**Pull request merged**